WooCommerce/WooCommerceTests/ViewRelated/Dashboard/StoreStats/StorePerformanceViewModelTests.swift

```diff
@@ -11,7 +11,7 @@ final class StorePerformanceViewModelTests: XCTestCase {
     @MainActor
     func test_dates_for_custom_range_are_correct_for_non_custom_time_range() throws {
         // Given
-        let viewModel = StorePerformanceViewModel(siteID: 123, usageTracksEventEmitter: .init())
+        let viewModel = StorePerformanceViewModel(siteID: 123, siteTimezone: .current, usageTracksEventEmitter: .init())
```

Comment:
👋 Just a note that this `test_dates_for_custom_range_are_correct_for_non_custom_time_range` and `TopPerformersDashboardViewModelTests.test_dates_for_custom_range_are_correct_for_non_custom_time_range` below are still failing at certain time (failed [3 times](https://buildkite.com/automattic/woocommerce-ios/builds/25938#01932970-a255-40b0-928c-05b8b5033d75) 7-8AM GMT, but fixed a few hours later).

cc @selanthiraiyan

Reply:
Logged issue in https://github.com/woocommerce/woocommerce-ios/pull/14287#discussion_r1843059760

PR to temporarily exclude them in https://github.com/woocommerce/woocommerce-ios/pull/14426